packages/gatsby-theme-newrelic/gatsby/on-client-entry.js

```diff
@@ -0,0 +1,21 @@
+import getLocale from '../src/utils/getLocale';
+
+export const onClientEntry = (_, themeOptions) => {
```

Comment:
It looks like [you're looking for the `default` export](https://github.com/newrelic/gatsby-theme-newrelic/pull/270/files#diff-ae38556db84ac9f7923212ab9135c98208a4fdcd41f6764e43fe41641b57abbcR20), but here you're exporting this as a named export. Can you instead convert this to a default export?

```js
const onClientEntry = () => ...

export default onClientEntry
```